json-endpoints/class.wpcom-json-api-list-post-types-endpoint.php

```diff
@@ -9,6 +9,8 @@ class WPCOM_JSON_API_List_Post_Types_Endpoint extends WPCOM_JSON_API_Endpoint {
 		'map_meta_cap' => 'map_meta_cap',
 		'cap'          => 'capabilities',
 		'hierarchical' => 'hierarchical',
+		'show_in_ui'   => 'show_in_ui',
```

Comment:
I noticed the `show_in_ui` value is coming back from the API as `null`. Should we make that return as a boolean?